ibm/mas_devops/roles/ocp_config/tasks/main.yml

```diff
@@ -25,3 +25,7 @@
 # -----------------------------------------------------------------------------
 - include_tasks: update-operatorhub.yml
   when: ocp_operatorhub_disable_redhat_sources
+
+# 5. Enable dual stack for ipv6 test
```

Comment:
I'd like to keep it there in case we still need it in the future. Currently it seems like this step is one-time deal only, if you run it manually once you don't need to do it again. But in case things change in the future I'd like to come back to this.

Reply:
Well it is not a good practice to keep code commented-out. If you think it is really needed please add more context to it.

Reply:
removed